aztec/src/main/kotlin/org/wordpress/aztec/spans/AztecOrderedListSpan.kt

```diff
@@ -55,7 +55,22 @@ class AztecOrderedListSpan(
         p.color = listStyle.indicatorColor
         p.style = Paint.Style.FILL
 
-        val lineIndex = getIndexOfProcessedLine(text, end)
+        var start = if (attributes.hasAttribute("start") == true) {
+            attributes.getValue("start").toInt()
+        } else {
+            0
+        }
+
+        val isReversed = attributes.hasAttribute("reversed")
+        var lineIndex = if (start > 0) {
+            if (isReversed) start - (getIndexOfProcessedLine(text, end) - 1)
+            else start + (getIndexOfProcessedLine(text, end) - 1)
+        } else {
+            val number = getNumberOfItemsInProcessedLine(text)
+            if (isReversed) number - (getIndexOfProcessedLine(text, end))
+            else getIndexOfProcessedLine(text, end)
+        }
+
         val textToDraw = if (lineIndex > -1) {
```

Comment:
I think we could remove this check. I tested on the Web and negative number are shown. Here is the diff:

```diff
-        val textToDraw = if (lineIndex > -1) {
-            if (dir >= 0) lineIndex.toString() + "."
+        val textToDraw = if (dir >= 0) lineIndex.toString() + "."
             else "." + lineIndex.toString()
-        } else {
-            ""
-        }
```

This is kind of weird if we only show the empty string as the prefix:

<img width="181" alt="Screenshot 2019-12-02 at 11 53 44" src="https://user-images.githubusercontent.com/40213/69954257-28ad1500-14fb-11ea-8a90-e06634602bf9.png">

Reply:
Hey @maxme, nice catch there. 
I have fixed it now and you can try when you have time :)